instrumentation/pg/test/fixtures/sql_table_name.json

```diff
@@ -50,5 +50,13 @@
     {
       "name": "from_with_join",
       "sql": "SELECT columns FROM test_table JOIN table2 ON test_table.column = table2.column"
+    },
+    {
+      "name": "table_name_with_single_quotes",
+      "sql": "SELECT columns FROM 'test_table'"
+    },
+    {
+      "name": "table_name_with_double_quotes",
+      "sql": "SELECT columns FROM \"test_table\""
```

Comment:
Would you kindly edit this file and add a trailing new line? It was missing in the original comment and should have been there.